dspace-spring-rest/pom.xml

```diff
@@ -155,6 +155,12 @@
                 <artifactId>spring-expression</artifactId>
                 <version>${spring.version}</version>
             </dependency>
+            <!-- Solr introduces multiple versions of zookeeper. So, we specify the version we want -->
```

Comment:
See dspace-solr/pom.xml.  If we now have two projects dependent on zookeeper, then we should pull both of these dependency management declarations up into a single one in parent.